core/tee/tee_svc_cryp.c

```diff
@@ -1162,11 +1303,16 @@ static TEE_Result tee_svc_cryp_obj_populate_type(
 	for (n = 0; n < attr_count; n++) {
 		size_t raw_size;
 		void *raw_data;
-		int idx =
-		    tee_svc_cryp_obj_find_type_attr_idx(attrs[n].attributeID,
+		int idx = tee_svc_cryp_obj_find_type_attr_idx(
+							attrs[n].attributeID,
 							type_props);
+		/* attribute not defined in current object type */
 		if (idx < 0)
-			continue;
+			return TEE_ERROR_ITEM_NOT_FOUND;
```

Comment:
panic reason: An attribute which is not defined for the object’s type is present in attrs